docs/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations_for_pandas.rst

```diff
@@ -3,17 +3,256 @@
 How to create custom Expectations for pandas
 ============================================
 
-.. admonition:: Admonition from Mr. Dickens
+Custom Expectations let you extend the logic for validating data to use any criteria you choose. This guide will show you how to extend the ``PandasDataset`` class with your own :ref:`Expectations`.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
+    - Installed Great Expectations (e.g. ``pip install great_expectations``)
+    - Have access to a notebook (e.g. ``jupyter notebook``, ``jupyter lab``, etc.)
+    - Be able to access data from your notebook
+    - Nothing else. Unlike most how-to guides, these instructions do *not* assume that you have configured a Data Context by running ``great_expectations init``.
```

Comment:
See note in other PR re: maintaining parallelism in bullets with "have already ... "

Reply:
Yep. Same thing here.

I'm open to the idea of replacing "Nothing else" with a grammatically parallel phrase. I couldn't think of anything off the top of my head, and it's one of those places where I'm okay with being deliberately mismatched for the sake of grabbing a jot of attention.

One alternative I considered was including a "no magic" icon, along with the standard text:

```
Unlike most how-to guides, these instructions do *not* assume that you have configured a Data Context by running ``great_expectations init``. For a list of other magic-free guides for Great Expectations, please visit `some link`.
```

We can do that if you prefer.

IMO, the "no-magic" intro is one of the most important pathways that we can create through our docs. It's clear that a substantial fraction of our user base wants to play with Expectations in notebooks before committing to a deployment using a Data Context.

I don't think we should try to change that, because it wouldn't work.

Instead, I want to accommodate magic-phobic users, let them get a taste of GE, and explain why a Data Context will be valuable as they reach the right moments in their journey.